prime-router/src/main/kotlin/fhirengine/azure/FHIRFunctions.kt

```diff
@@ -49,7 +51,15 @@ class FHIRFunctions(
         logger.info(
             "message consumed from elr-fhir-receive queue"
         )
-        process(message, dequeueCount, FHIRReceiver(), ActionHistory(TaskAction.receive))
+        process(message, dequeueCount, FHIRConverter(), ActionHistory(TaskAction.convert))
```

Comment:
The function name is receive-fhir, but it's the convert action...?

Reply:
I still need to clean this up.... but I'll be changing function name